morphir/src/org/finos/morphir/ir/distribution/Distribution.scala

```diff
@@ -56,5 +72,33 @@ object Distribution {
         dependencyPackageName: PackageName,
         dependencyPackageSpec: UPackageSpecification
     ): Distribution = Library(packageName, dependencies + (dependencyPackageName -> dependencyPackageSpec), packageDef)
+
+    def toLib: Lib = Lib(dependencies, packageDef)
   }
+
+  def toBundle(
+      packageName: PackageName,
+      dependencies: Map[PackageName, UPackageSpecification],
+      packageDef: PackageDefinition.Typed
+  ): Bundle = Bundle(Map(packageName -> Lib(dependencies, packageDef)))
+
+  def toBundle(packageName: PackageName, lib: Lib): Bundle = Bundle(Map(packageName -> lib))
+  def toBundle(dists: Distribution*): Bundle               = Bundle(toLibsMap(dists: _*))
+
+  def toLibrary(
+      packageName: PackageName,
+      dependencies: Map[PackageName, UPackageSpecification],
+      packageDef: PackageDefinition.Typed
+  ): Library = Library(packageName, dependencies, packageDef)
+
+  def toLibrary(packageName: PackageName, lib: Lib): Library = Library(packageName, lib.dependencies, lib.packageDef)
+  def toLibraries(dists: Distribution*): List[Library] = toLibsMap(dists: _*)
+    .map { case (packageName, lib) => toLibrary(packageName, lib) }
+    .toList
+
+  def toLibsMap(dists: Distribution*): Map[PackageName, Lib] =
+    dists.flatMap {
+      case (library: Library) => List(library.packageName -> library.toLib)
+      case (bundle: Bundle)   => bundle.libraries.toList
+    }.toMap
```

Comment:
A possible issue here is the package name collisions (the first one on the Distributins list will be lost).  Any specific way we should deal with this?